### ISSUE TITLE
Keyframes not bubbling to top

### ISSUE THREAD

**Issue by pdufour** [sass/libsass#472]
If you put a `@keyframes` block inside a selector, it nests, instead of bubbling to the top. Allow me to demonstrate.

libsass:
https://gist.github.com/pdufour/7d4f4e9eef53c3a22098

ruby sass:
https://gist.github.com/pdufour/fee5265b1b0e6f66a06c

I would try to fix this myself but unfortunately I don't know C or C++.

**Comment by mgreter** [comment]
I guess this should be the test case:

Input.css
```css
div {
  display: block;
  @keyframes {
    from {
      foo: bar;
    }
  }
}
```

Expected output (sass 3.4.9)
```
div {
  display: block; }
@keyframes {
  from {
    foo: bar; } }
```

Actual libsass output:
```
div {
  display: block;
  @keyframes {
    from {
      foo: bar; } }
 }
```

So this seems a valid bug. Maybe someone has the time to convert the above into a spec test!?

**Comment by xzyfer** [comment]
I'll address this with my bubbling work.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/243.

**Comment by xzyfer** [comment]
This is fixed and will be in 3.2.

**Comment by pdufour** [comment]
Thanks a lot!
